activities/communication/run-a-community-call.md

```diff
@@ -0,0 +1,109 @@
+* the notetaker
+
+## Before the call
+
+The chair and the notetaker:
+
+* create a notes HackMD.io document for this call
+* add "updates since the last community call" to the notes - keep this to three items or less (we want the community call to be a dialogue, not a broadcast). If possible, focus on community related updates
+* put these in the call notes doc
+
+Ask the [communications coordinator](../../organization/staff.md#communications) if you need help with Foundation updates.
+
+The notetaker will take notes during the call in the same document.
+
+### Conference call etiquette
+
+Each person should use their own computer so everyone on the call is symbolically equal.
+
+The chair should be on the call 5 minutes before the call starting time.
+
+Consider putting a Foundation for Public Code poster on the wall behind you.
+
+## Templated agenda items
+
+1. Opening the call
+
+* Introduction round (all)
+* Short update from the Foundation for Public Code (chair)
+
+2. Substance of the call
+
+## 1. Opening the call
+
+Call-in link: [https://meet.google.com/tfd-acmn-tkb[(https://meet.google.com/tfd-acmn-tkb)
+
+Agenda link: [https://hackmd.io/-OegeqvoThCbAsw3c3gIjw?edit](https://hackmd.io/-OegeqvoThCbAsw3c3gIjw?edit)
+
+### No-show
+
+If nobody joins at the call starting time, wait at least 15 minutes before closing the call. If possible, leave the call open the entire hour for potential late drop-ins.
+
+### Introduction round
+
+The chair:
+
+* welcomes everyone to the community call on behalf of the Foundation for Public Code
+* briefly mentions what is on the agenda for today (without being so specific that discussions start)
+* notes that the call will be minuted, but not recorded, and the minutes published on our blog. Everyone on the call will get the chance to review and fix the minutes before they're published
+* asks everyone to introduce themselves - who are you, do you represent an organization, what's your interest in public code or the topic for this call?
+* starts with themselves
+* asks someone else to go next (for example, someone they know to be excited about the topic, or a friend of the Foundation, or an extrovert)
+* keeps track of who's introduced themselves and nudges anyone who hasn't by the end
+
+### Short update from the Foundation for Public Code
+
+The chair:
+
+* thanks everyone for introducing themselves
+* asks if everyone's familiar with the Foundation
+* gives optional Foundation intro
+* gives update since the last community call (from the notes document)
+
+> #### Optional intro to the Foundation
+
+> The Foundation for Public Code is a member-driven, member-funded and member-governed association of public organizations.
+
+> We work to enable open and collaborative public-purpose software for public organizations internationally.
+
+> The Foundation for Public Code supports software at the codebase level through codebase stewardship and by publishing the Standard for Public Code.
```

Comment:
These are 4 different block-quotes. I doubt that this was even meant as being one quote, let alone 4 different ones.

If for instance we would like to make a print design where all block quotes have large `"` before and larger type, it this might not make sense to be highlighted that way.

I recommend stripping the block quotes.